src/public/sass/patterns/_related-items.scss

```diff
@@ -0,0 +1,12 @@
+// This is a GOV.UK Publishing specific component that
+// can be seen at http://govuk-static.herokuapp.com/component-guide/related_items
```

Comment:
The link did not work for me. Shouldn't that be https://components.publishing.service.gov.uk/component-guide/related_navigation?

Reply:
I don't think that is the correct link. the comments were taken from the prototype and since are out of date it is probably worth deleting.